config/prow/cluster/monitoring/mixins/Makefile

```diff
@@ -20,7 +20,7 @@ SHELL=/bin/bash -o pipefail
 all: grafana-dashboards prow_prometheusrule.yaml
 
 grafana-dashboards:
-	for name in deck ghproxy hook plank prow sinker tide; do echo "Generating $${name}.json ..."; jsonnet -J vendor/ -J lib/ "grafana_dashboards/$${name}.jsonnet" > "./dashboards_out/$${name}.json"; done
+	for name in boskos-http boskos deck ghproxy hook plank prow sinker slo tide; do echo "Generating $${name}.json ..."; jsonnet -J vendor/ -J lib/ "grafana_dashboards/$${name}.jsonnet" > "./dashboards_out/$${name}.json"; done
```

Comment:
Instead of an explicit list can we do something like `grafana_dashboards/*.jsonnet` to automatically build any files that are present?